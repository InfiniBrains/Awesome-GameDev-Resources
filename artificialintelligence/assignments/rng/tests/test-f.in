6 30 0 200
