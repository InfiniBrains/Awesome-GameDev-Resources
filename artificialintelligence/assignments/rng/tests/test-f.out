144
50
128
153
185
101
153
132
168
24
124
5
53
38
137
58
148
43
121
123
5
124
27
151
153
145
170
147
33
4
